source/source.go

```diff
@@ -29,6 +29,7 @@ import (
 	"time"
 	"unicode"
 
+	sprig "github.com/go-task/slim-sprig"
```

Comment:
Please no import for a single func.

Reply:
@szuecs it includes several useful go template functions: https://github.com/go-task/slim-sprig/blob/master/functions.go

Would you rather not include these functions, and only keeping `isIPv4`?